docs/home/index.md

```diff
@@ -53,15 +53,8 @@ data management across multiple cloud providers.
 :padding: 2
 :class-title: sd-fs-5
 
-Want to dive deep into CrateDB? Explore the comprehensive reference manual.
-
-At the core of CrateDB Cloud sits CrateDB, the open-source, distributed, and
-scalable SQL database system designed for storing and analyzing massive amounts
-of data in real-time, even with complex queries.
-
-Based on Lucene and inherited from Elasticsearch/OpenSearch, CrateDB supports
-time series, documents, and vectors, combining the simplicity of SQL with the
-scalability of a distributed architecture.
+Explore our comprehensive reference manual for detailed information and guidance
+on using CrateDB, the open-source database system at the core of CrateDB Cloud.
```

Comment:
Doing a bit of name dropping here, in order to feed search engine crawlers, was intentional. Any chance to bring those words back in one way or another?

Reply:
Re-added with b15208feb.